README.md

```diff
@@ -67,6 +67,24 @@ require('path/to/smtpapi-php/smtpapi-php.php');
 
 Previous versions of the library can be found in the [version index](https://sendgrid-open-source.s3.amazonaws.com/index.html).
 
+## Setup Environment Variables
+
+### Initial Setup
+
+```bash
+cp .env_sample .env
```

Comment:
Same thing as on the other ones - please remove the next section, and add a mention about just editing the .env file